packages/web/utils/ensHelpers.ts

```diff
@@ -0,0 +1,10 @@
+import { ethers } from 'ethers';
+
+export const getAddressFromName = async (ens: string) => {
+  // getting a provider without user connecting, todo: use whatever rpc metagame wants
+  const mainnetProvider = new ethers.providers.JsonRpcProvider(
+    'https://eth.llamarpc.com',
```

Comment:
```suggestion
    CONFIG.mainnetRPCURL,
```

Reply:
Do we already have a mainnetRPC to use, or should I make a new one?

Reply:
There is not one currently, you'll need to add it. It should pull from an environment variable `NEXT_PUBLIC_MAINNET_RPC` *(I think. Is is accessed from Next?)*.

Reply:
Created a new env for NEXT_PUBLIC_MAINNET_RPC and added in config.